src/data/navigation/header.js

```diff
@@ -18,6 +18,10 @@ module.exports = [
           title: "Checkout starter kit",
           path: "/starter-kit/checkout/"
         },
+        {
```

Comment:
Tax is part of the Checkout Starter Kit, along with Payment and Carrier integrations, so I believe we don't need a dedicated menu for it in the current structure.